fastlane/lanes/localization.rb

```diff
@@ -264,10 +260,9 @@
       File.join(download_path, 'en-US')
     )
 
-    # TODO: Remove this call after a few sprints of validating that that the `skip_upload_metadata: options[:track] != 'production'` recently added to `upload_build_to_play_store` lane works well
-    locales_including_enUS = [['en-gb', 'en-US']] + JP_RELEASE_NOTES_LOCALES # first item (GlotPress locale) unused for this action; second param = google_play locale
-    android_create_xml_release_notes(download_path: download_path, build_number: "#{options[:build_number]}", locales: locales_including_enUS)
-    sh("git add #{download_path} && git commit -m \"Update Jetpack metadata translations for #{options[:version]}\" && git push origin HEAD")
+    git_add(path: download_path)
+    git_commit(message: "Update Jetpack metadata translations for #{options[:version]}", allow_nothing_to_commit: true)
```

Comment:
See comment above

```suggestion
    git_commit(path: download_path, message: "Update Jetpack metadata translations for #{options[:version]}", allow_nothing_to_commit: true)
```

Reply:
I wanted to make sure my suggestion was correct so I tested it locally, might as well open a PR for it while at it: https://github.com/wordpress-mobile/WordPress-Android/pull/16290.

Reply:
Addressed by merging your PR #16290 into this branch via a1af416 🙂 🙇